components/generic/RelatedChip.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <b-badge
+    v-if="title"
+    :to="linkGen(id, title)"
+    pill
+    variant="light"
+    class="mt-1 mr-2 font-weight-normal bg-white"
+    :class="{ 'img-chip': img }"
+    data-qa="browse chip"
+  >
+    <b-img
+      v-if="img"
+      :src="img"
+      alt=""
+      rounded="circle"
+      class="mr-2"
+    />
+    {{ title }}
+  </b-badge>
+</template>
+
+<script>
+  export default {
+    name: 'RelatedChip',
+
+    props: {
+      linkGen: {
+        type: Function,
+        default: (val) => val
+      },
+      title: {
+        type: String,
+        default: () => {}
```

Comment:
If there is no title, then nothing is rendered. In other words, title is required. Make it so in the prop definition and remove the v-if on the b-badge.